### PR TITLE
Use of conda to simplify Jekyll and module installation

### PR DESCRIPTION
**Pull request by bebatut**
Hi,

On the advise of @bgruening, we can use conda to manage the installation Jekyll and its modules.
It can then simplified the Makefile. 

I documented that in the dedicated tutorial. In the same time, I simplified the CONTRIBUTING.md and the README.md to link to the "Train the trainers" tutorials where everything needed to develop a tutorial is explained.

Bérénice

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -126,265 +116,21 @@ For each topic, a flavored Docker image must integrate the tools needed for
...
-        ```
-        > ### :pencil2: Hands-on:
-        >
-        > 1. **Sort BAM dataset** :wrench:: Sort the paired-end BAM file by "Read names" with **Sort BAM dataset**
-        {: .hands_on}
-        ```
+The `docker` image must also integrate a Galaxy tour from the [`tours` repository](https://github.com/galaxyproject/galaxy-tours)
 
-        ![](shared/images/tutorial_hand_on_box.png)
+> Want to learn more to a tutorial? [Check out our tutorial to build a Docker flavor for a tutorial](https://galaxyproject.github.io/training-material//topics/training/tutorials/create-new-tutorial-docker/tutorial.html)
```
Comment (martenson) [inline CONTRIBUTING.md]: >Want to learn more to a tutorial?

Seems like a typo. Add?

File: environment.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: galaxy_training_material
+channels:
+- bioconda
+- r
```
Comment (bgruening) [inline environment.yml]: r is not needed.
conda-forge should be before defaults.
actually I'm not sure bioconda is needed :)

Comment (bebatut) [inline environment.yml]: I changed that

File: CONTRIBUTING.md (hunk 1 above)
Comment (bebatut) [inline CONTRIBUTING.md]: Fixed! Thanks!

**Pull request merged**